server/src/routes/api/v1/showRouter.js

```diff
@@ -0,0 +1,25 @@
+import express from "express"
+// import { Show } from 
+
+const showsRouter = new express.Router()
+
+showsRouter.get("/:id/info", async(req,res) => {
```

Comment:
this path is technically fine, but ideally should match convention where possible, which means just 

```
showsRouter.get("/:id", async (req,res) => {
```